docs/ruleset_functions/get_spaces_served_by_SWH_use.md

```diff
@@ -17,12 +17,17 @@ Data Lookup: None
 
 Logic:
 
+- create a list of spaces: `spaces_served = []`
 - find the building_segment that contains this swh_use by looking through each building segment: `for building_segment in RMD.building_segments:`
     - check if the swh_use is in this building segment: `if swh_use in building_segment.service_water_heating_uses:`
```

Comment:
If we want to check if swh_uses object is under the building segment level, should this line be ```if bldg_segment.get("service_water_heating_uses"):```?